sample/build.gradle

```diff
@@ -57,6 +57,18 @@ android {
         exclude 'META-INF/gradle/incremental.annotation.processors'
     }
 }
+if (project.hasProperty("useKsp")) {
+    ksp {
+        arg("skipPrivate", "true")
```

Comment:
is there a better naming convention I should follow for annotation processor options that are passed by the user?